jscomp/test/record_extension_test.ml

```diff
@@ -0,0 +1,27 @@
+
+#if OCAML_VERSION =~ ">4.3.0" then
+
+(* Record_extension *)
+type t0  = ..
+type t0 += Inline_record of { x : int ; y : string} 
+
+let f x = 
+  match x with 
+  | Inline_record {x; y} -> Some (x + int_of_string y)  
+  | _ -> None 
+
+(* Record_unboxed *)
+type t1 = | A of { x : int } [@ocaml.unboxed]
```

Comment:
Looks like `t1` is never used.

Reply:
Yes, the unboxed was never triggered. 
I intended to create an example trigger `Record_unboxed false`, `Record_unboxed true`